domain/src/Services/TransactionDispatcher/Handlers/IncomeHandler.php

```diff
@@ -26,7 +26,8 @@ public function handle(Transaction $transaction): void
         $taxYearId = TaxYearId::fromTaxYear($taxYear);
         $taxYearAggregate = $this->taxYearRepository->get($taxYearId);
 
-        $taxYearAggregate->recordIncome(new RecordIncome(taxYear: $taxYear, amount: $transaction->costBasis));
+        // @phpstan-ignore-next-line
```

Comment:
ℹ️ Starting to use these more, because most of them will be flagged as redundant by PHPStan once the planned refactorings are done